packages/formik-date/src/Date.js

```diff
@@ -0,0 +1,174 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { useField, useFormikContext } from 'formik';
+import moment from 'moment';
+import { useToggle } from '@availity/hooks';
+import 'react-dates/initialize';
+import { SingleDatePicker } from 'react-dates';
+import 'react-dates/lib/css/_datepicker.css';
+import './css/react-dates-overrides.css';
+import Icon from '@availity/icon';
+import { FormGroup, Input, Feedback } from '@availity/form';
+
+const isOutsideRange = (min, max) => day => {
+  if (moment.isMoment(min) && moment.isMoment(max)) {
+    return day.isBefore(min) || day.isAfter(max);
+  }
+
+  if (typeof min === 'string' && typeof max === 'string') {
+    return day.isBefore(moment(min)) || day.isAfter(moment(max));
+  }
+
+  if (
+    min &&
+    min.value !== undefined &&
+    min.units &&
+    max &&
+    max.value !== undefined &&
+    max.units
+  ) {
+    const start = moment().subtract(min.value, min.units);
+    const end = moment().add(max.value, max.units);
+    return day.isBefore(start) || day.isAfter(end);
+  }
+
+  return false;
+};
+
+const AvDate = ({
+  label,
+  value: valueFromProps,
```

Comment:
I'm not sure how passing in the `value` prop is going to play with formik. Since its managing the `value` state with the metadata it could possibly have a different value than what the `value` prop of the field has.